roles/kubernetes-apps/policy_controller/tasks/main.yml

```diff
@@ -17,4 +19,7 @@
     filename: "{{kube_config_dir}}/calico-policy-controller.yml"
     namespace: "{{system_namespace}}"
     resource: "rs"
-  when: inventory_hostname == groups['kube-master'][0]
+  when: inventory_hostname == groups['kube-master'][0] and
```

Comment:
IMHO delegate_to: "{{groups['kube-master'][0]}}"
will allow to have less complex when condition